script.js

```diff
@@ -0,0 +1,41 @@
+const result = document.querySelector('#app')
+const ul = document.getElementById('#results')
```

Comment:
If your using getElementById then you don't need to # before results.
Change to:
document.getElementById('results') or document.querySelector('#results')